landing-page/src/components/landing-page-network.ts

```diff
@@ -0,0 +1,163 @@
+import "@material/mwc-linear-progress/mwc-linear-progress";
+import {
+  type CSSResultGroup,
+  LitElement,
+  type PropertyValues,
+  css,
+  html,
+  nothing,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import type {
+  LandingPageKeys,
+  LocalizeFunc,
+} from "../../../src/common/translations/localize";
+import "../../../src/components/ha-button";
+import "../../../src/components/ha-alert";
+import {
+  ALTERNATIVE_DNS_SERVERS,
+  getSupervisorNetworkInfo,
+  setSupervisorNetworkDns,
+} from "../data/supervisor";
+import { fireEvent } from "../../../src/common/dom/fire_event";
+
+const SCHEDULE_FETCH_NETWORK_INFO_SECONDS = 5;
+
+@customElement("landing-page-network")
+class LandingPageNetwork extends LitElement {
+  @property({ attribute: false })
+  public localize!: LocalizeFunc<LandingPageKeys>;
+
+  @state() private _networkIssue = false;
+
+  @state() private _getNetworkInfoError = false;
+
+  @state() private _dnsPrimaryInterface?: string;
+
+  protected render() {
+    if (!this._networkIssue && !this._getNetworkInfoError) {
+      return nothing;
+    }
+
+    if (this._getNetworkInfoError) {
+      return html`
+        <ha-alert alert-type="error">
+          <p>${this.localize("network_issue.error_get_network_info")}</p>
+        </ha-alert>
+      `;
+    }
+
+    return html`
```

Comment:
Should we render this if we couldnt fetch the supervisor info?

Reply:
:thinking: we don't render it if there is `_getNetworkInfoError  = true`